{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -104,6 +104,9 @@ vulture = "^2.7"
 {%- if cookiecutter.use_mccabe in ['yes', 'y'] %}
 mccabe = "^0.6.1"
 {%- endif %}
+{%- if cookiecutter.add_containers in ['Docker', 'Podman'] %}
+compose-go = "^1.0.4"
```

Comment:
maybe it would be nice to add a note here
something like:
`# if you want to use docker-compose from your system, remove compose-go here`